packages/desktop-client/src/components/FinancesApp.js

```diff
@@ -77,49 +77,69 @@ function PageRoute({ path, component: Component }) {
   );
 }
 
-function Routes({ isMobile, location }) {
+function Routes({ location }) {
+  const { isNarrowWidth } = useViewport();
   return (
     <Switch location={location}>
       <Route path="/" exact render={() => <Redirect to="/budget" />} />
 
-      <PageRoute path="/reports" component={Reports} />
-      <PageRoute path="/budget" component={isMobile ? MobileBudget : Budget} />
-
-      <Route path="/schedules" exact component={Schedules} />
-      <Route path="/schedule/edit" exact component={EditSchedule} />
-      <Route path="/schedule/edit/:id" component={EditSchedule} />
-      <Route path="/schedule/link" component={LinkSchedule} />
-      <Route path="/schedule/discover" component={DiscoverSchedules} />
-      <Route
-        path="/schedule/posts-offline-notification"
-        component={PostsOfflineNotification}
-      />
-
-      <Route path="/payees" exact component={ManagePayeesPage} />
-      <Route path="/rules" exact component={ManageRulesPage} />
-      <Route path="/settings" component={Settings} />
-      <Route path="/nordigen/link" exact component={NordigenLink} />
-
-      <Route
-        path="/accounts/:id"
-        exact
-        children={props => {
-          const AcctCmp = isMobile ? MobileAccount : Account;
-          return (
-            props.match && <AcctCmp key={props.match.params.id} {...props} />
-          );
-        }}
-      />
-      <Route
-        path="/accounts"
-        exact
-        component={isMobile ? MobileAccounts : Account}
-      />
+      {isNarrowWidth ? (
```

Comment:
Didn't love the duplication here, but it allows us to smoothly redirect when a user resizes to a view too narrow to support the screen they're on (like Reports or Schedules). Maybe there's a better way. This change is an isolated commit.